events-processor/processors/processors.go

```diff
@@ -114,15 +135,11 @@ func StartProcessingEvents() {
 		panic(err.Error())
 	}
 
-	maxConns := 200
-	maxConnsStr := os.Getenv("LAGO_EVENTS_PROCESSOR_DATABASE_MAX_CONNEXIONS")
-	if maxConnsStr != "" {
-		value, err := strconv.Atoi(maxConnsStr)
-		if err != nil {
-			logger.Error("Error converting max connections into integer", slog.String("error", err.Error()))
-		}
-
-		maxConns = value
+	maxConns, err := utils.GetEnvAsInt("LAGO_EVENTS_PROCESSOR_DATABASE_MAX_CONNEXIONS", 200)
```

Comment:
`LAGO_EVENTS_PROCESSOR_DATABASE_MAX_CONNECTIONS` is preferable I think

Reply:
Ahhh yes, good catch, thanks!